app.js

```diff
@@ -109,10 +109,18 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function multiplyArray(testArray){ //eslint-disable-line
+  var a = testArray[0];
+  var b = testArray[1];
+  var c = testArray[2];
 
+  var condenseProduct = multiply(a, b)[0];
+  var finalProduct = multiply(condenseProduct, c)[0];
+  var productString = 'The numbers ' + a + ',' + b + ',' + c + ' have a product of ' + finalProduct + '.';
+
+  return [finalProduct, productString];
 }
```

Comment:
Technically this is perfect, however it works if and _only_ if the array passed in is three items long. If it is any shorter or longer it no longer works. Same for your `sumArray` function.